lib/Parser/ptree.h

```diff
@@ -856,55 +919,33 @@ class ParseNodeSpecialName : public ParseNodePid
 class ParseNodeSuperReference : public ParseNodeBin
 {
 public:
+    ParseNodeSuperReference(OpCode nop, charcount_t ichMin, charcount_t ichLim, ParseNode * pnode1, ParseNode * pnode2);
+
     ParseNodePtr pnodeThis;
 };
 
 // call node with super target
 class ParseNodeSuperCall : public ParseNodeCall
 {
 public:
+    ParseNodeSuperCall(OpCode nop, charcount_t ichMin, charcount_t ichLim, ParseNode * pnodeTarget, ParseNode * pnodeArgs);
+
     ParseNodePtr pnodeThis;
     ParseNodePtr pnodeNewTarget;
+};
+
+#define AssertNodeMem(pnode) AssertPvCb(pnode, sizeof(ParseNode))
+#define AssertNodeMemN(pnode) AssertPvCbN(pnode, sizeof(ParseNode))
+
```

Comment:
The definitions I've found for AssertPvCb and AssertPvCbN in Common\Core\Assertions.h don't seem to be useful (AssertPvCbN is noop and AssertPvCb would never fail when used like here). Do we need to keep AssertNodeMem and AssertNodeMemN? Did they use to be more meaningful in the past?

Reply:
It was meaningful in the past, and not any longer.  I intend to clean it up int he next change.